src/node_state/verifiers.rs

```diff
@@ -95,10 +95,16 @@ pub fn risc0_proof_verifier(encoded_receipt: &[u8], image_id: &[u8]) -> Result<H
 
     let image_bytes: Digest = image_id.try_into().expect("Invalid Risc0 image ID");
 
-    match receipt.verify(image_bytes) {
-        Ok(_) => (),
-        Err(e) => bail!("Risc0 proof verification failed: {}", e),
-    };
+    let dev_mode = std::env::var("RISC0_DEV_MODE").unwrap_or_else(|_| "0".to_string());
+
+    if dev_mode == "1" {
```

Comment:
En lisant la doc, il me semble qu'on peut garder la fonction de vérification qu'on a sans rien changé:
```
A risc0 project, when run in dev-mode by setting the RISC0_DEV_MODE environment variable, supports ([fake](https://docs.rs/risc0-zkvm/1.1/risc0_zkvm/enum.InnerReceipt.html#variant.Fake)) receipt creation and pass-through 'verification' function, so that dev-mode may be switched on and off at runtime without impacting project workflows.
```

Reply:
oui indeed, mais quand j'ai essayé ça a pas marché :/ je re-test

Reply:
bon en effet ça marche sans ces lignes... j'update la PR